rudder-agent/SOURCES/Makefile.in

```diff
@@ -60,8 +60,8 @@ jq_SHA256 = 1b3906e0c6dea26b5b9dcf864cc7334522f520d69543043c64b2c487e0c663ca
 cfengine_RELEASE = 3.18.3
 cfengine_SHA256 = 6b7545bb0c7dc556aca713664f5d6f4af13e2f9d9585f0f07d5785ef03e3d575
 # Original URL: http://www.cpan.org/src/5.0/perl-5.22.4.tar.gz
-perl_RELEASE = 5.37.5
-perl_SHA256 = b52bed9b0f7c039367b51cd61ae1bbe89d6d93ad8b2ed9d35760b0d7e80fb93e
+perl_RELEASE = 5.36.0
+perl_SHA256 = e26085af8ac396f62add8a533c3a0ea8c8497d836f0689347ac5abd7b7a4e00a
```

Comment:
Comment for original url is not correct (it wasn't before, but we should correct it nonetheless)